ios/Classes/PTFlutterViewController.m

```diff
@@ -310,12 +336,27 @@ - (void)applyReadOnly
     self.thumbnailsViewController.editingEnabled = !self.readOnly;
 }
 
-- (void)setAnnotationAuthor:(NSString *)annotationAuthor {
-    self.toolManager.annotationAuthor = [annotationAuthor copy];
+- (void)applyNavIcon
+{
+    if (self.showNavButton) {
+        UIImage *navImage = [UIImage imageNamed:self.navButtonPath];
+        UIBarButtonItem *navButton;
+        if (navImage == nil) {
+            navButton = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemDone target:self action:@selector(topLeftButtonPressed:)];
+        } else {
+            navButton = [[UIBarButtonItem alloc] initWithImage:navImage
+                                                         style:UIBarButtonItemStylePlain
+                                                        target:self
+                                                        action:@selector(topLeftButtonPressed:)];
+        }
+        self.navigationItem.leftBarButtonItem = navButton;
+    }
 }
 
-- (void)setContinuousAnnotationEditing:(BOOL)continuousAnnotationEditing {
-    self.toolManager.tool.backToPanToolAfterUse = !continuousAnnotationEditing;
+- (void)setThumbnailViewEditingEnabled:(BOOL)thumbnailViewEditingEnabled
+{
+    _thumbnailViewEditingEnabled = thumbnailViewEditingEnabled;
+    self.thumbnailViewEditingEnabledHasBeenSet = YES;
```

Comment:
@Maxwell-Yang-PDFTron maybe we can meet this afternoon to discuss. What I was trying to say before, this would be implemented as:

```
- (void)setThumbnailViewEditingEnabled:(BOOL)thumbnailViewEditingEnabled
{
    self.thumbnailsViewController.editingEnabled = thumbnailViewEditingEnabled;
}
```

and the getter would be:

```
- (BOOL)thumbnailViewEditingEnabled
{
    return self.thumbnailsViewController.editingEnabled;
}
```

What this is doing, is that the `PdftronFlutterPlugin`'s property `thumbnailViewEditingEnabled` stores no information at all - all it does is pass information back and forth between the caller and its composed `self.thumbnailsViewController.editingEnabled`. Does that make sense? Is there a reason not to do this?